main.go

```
@@ -47,430 +34,88 @@ func NewLogger() (*Logger, error) {
 	}
 	fmt.Println("Startup notification sent successfully")
 
-	light := Light{}
+	// Initialize the light with automatic detection
+	light, cleanup, err := initializeLight(logger)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer cleanup()
+
 	light.Clear()
```

golangci-lint (1.62.2)

Error return value of `light.Clear` is not checked

(errcheck)